spec/factory_helpers.rb

```diff
@@ -0,0 +1,8 @@
+module FactoryHelpers
```

Comment:
a good alternative for this would be using [Faker](https://github.com/faker-ruby/faker)